hazelcast/src/main/java/com/hazelcast/nio/IOUtil.java

```diff
@@ -275,17 +275,17 @@ public static Object readAttributeValue(ObjectDataInput in) throws IOException {
     }
 
     /**
-     * Closes the Closable quietly. So no exception will be thrown. Can also safely be called with a null value.
-     *
-     * @param closeable the Closeable to close.
+     * Quietly attempts to close a {@link Closeable} resource, swallowing any exception.
+     * @param closeable the resource to close. If {@code null}, no action is taken.
      */
     public static void closeResource(Closeable closeable) {
-        if (closeable != null) {
-            try {
-                closeable.close();
-            } catch (IOException e) {
-                Logger.getLogger(IOUtil.class).finest("closeResource failed", e);
-            }
+        if (closeable == null) {
```

Comment:
Just a bit simpler code. I did it while tracing through some code and then later didn't revert as it seemed as an improvement, even if very minor one. If  there's any reason not to do such minor changes, i'll revert it.